.jshintrc

```diff
@@ -1,7 +1,7 @@
 {
   "asi": true,
   "laxcomma": true,
-  "es5": true,
+  "esversion": 6,
```

Comment:
This is fine for master but shouldn't be backported to 3.x (do we have the equivalent of core's dont-land labels)?

Reply:
I don't see labels for this. Didn't we already do the final v3 release? I was expecting the next one to be v4 only, though we could do v3 if there is a pressing need of some sort.

This would always be semver-major defensively. Because this is written using new javascript features and removes workarounds for old node versions, this is effectively semver-major.